src/Foundation/NSAction.cs

```diff
@@ -107,6 +107,8 @@ public void Fire (NSTimer timer)
 	// Use this for asynchronous operations
 	[Register ("__MonoMac_NSAsyncActionDispatcher")]
 	internal class NSAsyncActionDispatcher : NSObject {
+		public const string SelectorName = "xamarinApplySelector";
+		public static readonly Selector Selector = new Selector (SelectorName);
```

Comment:
the `Selector` should beside a `#if MONOMAC` since it's not used for XI

Reply:
👍 Will fix others too.